src/DDTrace/Integrations/Swoole/SwooleIntegration.php

```diff
@@ -141,6 +149,7 @@ public function init(): int
         \DDTrace\hook_method(
             'Swoole\Http\Server',
             '__construct',
+            null,
```

Comment:
@labbati Tests were failing when the hook was changed from being a pre-hook to a post-hook